packit_service/worker/build/build_helper.py

```diff
@@ -373,9 +389,28 @@ def _report(
         The status reporting should be done through this method
         so we can extend it in subclasses easily.
         """
-        self.status_reporter.report(
-            description=description, state=state, url=url, check_names=check_names,
-        )
+        try:
+            self.status_reporter.report(
+                description=description, state=state, url=url, check_names=check_names,
+            )
+        except Exception as e:
```

Comment:
Catching such general exception can be dangerous -- there are a lot of other problems we can get.